teuthology-pull-requests/config/definitions/teuthology-pull-requests.yml

```diff
@@ -70,3 +70,6 @@
           !include-raw:
             - ../../../scripts/build_utils.sh
             - ../../build/build
+
+    publishers:
+      - github-notifier
```

Comment:
For pull requests to display something other than 'default' this needs to be added to the `github-pull-request` section:

    status-context: "custom-text-here"

And I don't see that has ever been set there to "teuthology-pull-requests". Is it possible that is coming from Travis?